### ISSUE TITLE
utils not installed with the module

### ISSUE THREAD

**Issue by Martexiwell** [mathieu17g/matplotlib_inline_labels#2]
When I installed the package by calling 
```python
pip install matplotlib-inline-labels
```
and ran 
```python
from inline_labels import add_inline_labels
```
I got
```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[2], line 1
----> 1 from inline_labels import add_inline_labels

File C:\Python\Python312\Lib\site-packages\inline_labels.py:1
----> 1 from utils import (
      2     # timer,
      3     esc,
      4     Timer,
      5 )
      6 from datatypes import (
      7     Line_Chunk_Geometries,
      8     Label_Rotation_Estimates_Dict,
   (...)
     11     Labels_PRcs,
     12 )
     13 from drawing import (
     14     get_dbg_axes,
     15     retrieve_lines_and_labels,
   (...)
     21     add_dbg_noninlined_labels_legend,
     22 )

ModuleNotFoundError: No module named 'utils'
```
... there is probably a problem somewhere (possibly on my side, I am not an expert).
My python installation is
```
> py -V
Python 3.12.3
> pip -V
pip 24.0 from C:\Python\Python312\Lib\site-packages\pip (python 3.12)
```

**Comment by mathieu17g** [comment]
Thank you for raising the issue. 

It has appeared when I splitted the code into several module (without using subpackages). I’m currently diving into it. 
I have a functional package on testpypi.

You can try it with:
```
pip install -i https://test.pypi.org/simple/ matplotlib-inline-labels==v0.2.1.22
```
I still have to solve some side issues with testing github action before releasing a patch.

**Comment by mathieu17g** [comment]
@Martexiwell it should be fixed now in v0.2.2, available on standard PyPI server
